cmd/controller/main.go

```diff
@@ -47,18 +46,22 @@ func main() {
 
 	op.
 		WithControllers(ctx, corecontrollers.NewControllers(
+			ctx,
+			op.Manager,
 			op.Clock,
 			op.GetClient(),
-			state.NewCluster(op.Clock, op.GetClient(), cloudProvider),
 			op.EventRecorder,
 			cloudProvider,
 		)...).
 		WithWebhooks(ctx, corewebhooks.NewWebhooks()...).
 		WithControllers(ctx, controllers.NewControllers(
 			ctx,
+			op.Manager,
 			op.GetClient(),
+			op.EventRecorder,
 			aksCloudProvider,
 			op.InstanceProvider,
 		)...).
-		Start(ctx)
+		//WithWebhooks(ctx, webhooks.NewWebhooks()...).
```

Comment:
Commented out? Should this be removed, or added?

Reply:
Just is a placeholder for where we would wire provider-specific webhooks (e.g. conversion for AKSNodeClass), would like to keep it as comment